api/src/test/groovy/uk/gov/dhsc/htbhf/claimant/controller/NewClaimSpec.groovy

```diff
@@ -140,6 +145,13 @@ class NewClaimSpec extends Specification {
         assertValidationResponse(response, "claimant.dateOfBirth", "must not be null")
     }
 
+    private assertAddress(Address actual, AddressDTO expected) {
```

Comment:
Not a great name for the method, doesn't say what it's asserting, should at least say `assertAddressValid` or similar.